package.json

```diff
@@ -1,9 +1,18 @@
 {
-  "name": "ca-project",
+  "name": "social-media-app",
   "version": "0.0.1",
-  "description": "project for school",
+  "description": "school project",
   "main": "index.js",
+   "dependencies": {
+    "boostrap": "^2.0.0"
+  },
+  "devDependencies": {
+    "bootstrap": "^5.2.1",
```

Comment:
```suggestion
```

Reply:
You should only have it in dependencies, not twice

Reply:
done now?